src/components/PeopleManagement/index.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useContext } from 'react';
+import { Helmet } from 'react-helmet';
+import { useIntl, FormattedMessage } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow, Button, Card, Icon, IconButtonWithTooltip, useToggle,
+} from '@openedx/paragon';
+import { Add, InfoOutline } from '@openedx/paragon/icons';
+
+import cardImage from './images/ZeroStateImage.svg';
+import Hero from '../Hero';
+import { SUBSIDY_TYPES } from '../../data/constants/subsidyTypes';
+import { EnterpriseSubsidiesContext } from '../EnterpriseSubsidiesContext';
+
+const PeopleManagementPage = () => {
+  const intl = useIntl();
+  const PAGE_TITLE = intl.formatMessage({
+    id: 'admin.portal.people.management.page',
+    defaultMessage: 'People Management',
+    description: 'Title for the people management page.',
+  });
+
+  const {
+    enterpriseSubsidyTypes,
+  } = useContext(EnterpriseSubsidiesContext);
+
+  const hasLearnerCredit = enterpriseSubsidyTypes.includes(SUBSIDY_TYPES.budget);
+  const hasOtherSubsidyTypes = enterpriseSubsidyTypes.includes(SUBSIDY_TYPES.license)
+    || enterpriseSubsidyTypes.includes(SUBSIDY_TYPES.coupon);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [isModalOpen, openModal, closeModal] = useToggle(false);
+
+  const tooltipContent = (
+    <FormattedMessage
+      id="admin.portal.people.management.page.tooltip.content"
+      defaultMessage="Only available for Learner Credit"
+      description="Tooltip to say this is only available for Learner Credit (not codes or subcriptions)."
+    />
+  );
+
+  return (
+    <>
+      <Helmet title={PAGE_TITLE} />
+      <Hero title={PAGE_TITLE} />
+      {hasLearnerCredit && (
```

Comment:
are we only intending for the people management page to show for customers with learner credits? if so, we should move this check up another layer. for customers without a learner credit plan, this page is showing up blank so it's unclear to the user if the page is intended to be blank or not. i.e. https://localhost.stage.edx.org:1991/adam-autoapplied-enterprise/admin/people-management

<img width="1512" alt="Screenshot 2024-09-30 at 2 35 57 PM" src="https://github.com/user-attachments/assets/644fe642-ff97-40f0-9f9a-2bb6c1d61269">

Reply:
follow-up question: the people management page still shows up empty for this customer. do we also need to add a check for if the customer has subscriptions? even though this customer doesn't have learner credit, they do have active subscriptions so we should still display the zero state card per figma.